packages/app/src/app/pages/Dashboard/Components/Selection/ContextMenus/ContainerMenu.tsx

```diff
@@ -13,7 +12,6 @@ export const ContainerMenu: React.FC<ContainerMenuProps> = ({
   createNewSandbox,
```

Comment:
I think we need to indicate in this typing that it can be null, for others readers

Reply:
Done!